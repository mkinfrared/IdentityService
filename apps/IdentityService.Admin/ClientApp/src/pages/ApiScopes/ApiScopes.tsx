import css from "./ApiScopes.module.scss";
import { ApiScopesProps } from "./ApiScopes.type";

const ApiScopes = ({ className }: ApiScopesProps) => {
  const classNames = [css.ApiScopes, className].filter(Boolean).join(" ");

  return (
    <div className={classNames} data-testid="ApiScopes">
      ApiScopes works!!
    </div>
  );
};

export default ApiScopes;
