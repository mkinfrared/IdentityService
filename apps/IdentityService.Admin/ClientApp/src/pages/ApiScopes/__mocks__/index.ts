export { default } from "./ApiScopes.mock";
