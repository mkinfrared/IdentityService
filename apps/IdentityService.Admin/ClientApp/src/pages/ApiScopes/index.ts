export { default } from "./ApiScopes";

export * from "./ApiScopes.type";
