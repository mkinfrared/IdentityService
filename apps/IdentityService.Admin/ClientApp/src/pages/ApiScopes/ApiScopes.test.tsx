import { render } from "@testing-library/react";

import ApiScopes from "./ApiScopes";

describe("<ApiScopes />", () => {
  const Component = <ApiScopes />;

  it("should be defined", () => {
    expect(ApiScopes).toBeDefined();
  });

  it("should match the snapshot", () => {
    const { container } = render(Component);

    expect(container).toMatchSnapshot();
  });

  it("should contain a data test id", () => {
    const { getByTestId } = render(Component);
    const element = getByTestId("ApiScopes");

    expect(element).toBeDefined();
  });
});
