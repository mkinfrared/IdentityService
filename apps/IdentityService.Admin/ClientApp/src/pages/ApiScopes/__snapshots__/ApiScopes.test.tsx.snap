// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ApiScopes /> should match the snapshot 1`] = `
<div>
  <div
    class="ApiScopes"
    data-testid="ApiScopes"
  >
    ApiScopes works!!
  </div>
</div>
`;
